### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by note**
Basically updated dependencies and added a sample for simple test.

### CODE REVIEW

File: src/main/g8/default.properties [hunk 1]
```diff
@@ -1,10 +1,9 @@
 description = Creates an empty Scala.js project.
 
-name = My Scala.js Project
+name = my-scala-js
```
Comment (note) [inline src/main/g8/default.properties:3]: Deliberately changed the default value to one does not containing `.` - was causing some problem. The issue was that created file was named `my-scala-js-project-fastopt.js` while index.html was trying to include `my-scala.js-project-opt.js`. I was not investigating that issue, just changed it here so at least it works for default values.

Comment (tues) [reply src/main/g8/default.properties:3]: True, I believe I even have a task titled "handle '.' in project name" somewhere on my todo list ;) I'll better open an issue instead.

**Comment by tues** [comment]
Thank you very much for your contribution :) I haven't had any opportunities to work with Scala.js in the past few months and as a result I neglected this project, so these changes are very welcome.

File: src/main/g8/default.properties (hunk 1 above)
Comment (tues) [reply src/main/g8/default.properties:3]: Opened #2

**Pull request merged**